wetube-api/src/main/kotlin/com/leeparkim/wetube/application/auth/OAuthApplicationService.kt

```diff
@@ -0,0 +1,32 @@
+package com.leeparkim.wetube.application.auth
+
+import com.leeparkim.wetube.domain.user.SocialType
+import org.springframework.stereotype.Service
+
+
+@Service
+class OAuthApplicationService(var googleOauth: GoogleOauth) {
+
+    fun getSocialAuth(socialLoginType: String): SocialOauth {
+        val socialOauth: SocialOauth = when (SocialType.valueOf(socialLoginType.uppercase())) {
+            SocialType.GOOGLE -> googleOauth
+            else -> {
+                throw IllegalArgumentException("알 수 없는 소셜 로그인 형식입니다.")
+            }
+        }
+
+        return socialOauth
+    }
+
+    fun getRedirectUrl(socialLoginType: String): String {
+        val socialOauth = getSocialAuth(socialLoginType)
+        val redirectURL: String = socialOauth.getOauthRedirectURL()
+        return redirectURL
```

Comment:
```suggestion
        return socialOauth.getOauthRedirectURL()
```
코틀린은 불필요한 변수할당은 지양하고 있어요. 이 방식이 괜찮아보여요